macro/o2sim.C

```diff
@@ -84,9 +84,6 @@ FairRunSim* o2sim_init(bool asservice)
   run->SetName(confref.getMCEngine().c_str()); // Transport engine
   run->SetIsMT(confref.getIsMT());             // MT mode
 
-  /** set external decayer **/
-  run->SetPythiaDecayer(TString(confref.getDecayConfig()));
```

Comment:
so this means we'll completely switch from Pythia6 to Pythia8 as decayer?

Reply:
I would rather say this means that we are getting in the o2 full control of the external decayer. 
Whether to use Pythia6 or Pythia8 can be discussed. I plan to develop the Pythia6 decayer as well.

I see advantages in using Pythia8, because it is maintained by authors and has up-to-date particle information and decay tables. Another advantage is that multiple instances of the class are allowed in the same program, namely one can have a Pythia8 generator running with completely different settings wrt. the Pythia8 decayer.

The disadvantage is that this creates a dependence to Pythia8 that I do not like very much.

The trouble with Pythia6 is if by any chance one runs a Pythia6 event generator simulation.
If both the generator and the simulation run in the same process (like in o2-sim-serial) both the Pythia6 generator and decayer will share the same fortran common blocks.